src/components/Avatar/AvatarMenu.js

```diff
@@ -18,6 +18,8 @@ const propTypes = {
         PropTypes.string,
     ]).isRequired,
     className: PropTypes.string,
+    onMenuClose: PropTypes.func,
+    onMenuOpen: PropTypes.func,
```

Comment:
I'm itching to ask you to change the props declaration to the flow type notation, but I will not do that here.

Reply:
Just realised this is ui-react! LOL